### ISSUE TITLE
No deploy option when creating project under non-default directory

### ISSUE THREAD

**Issue by JesusFreke** [malware-dev/MDK-SE#179]
When I create a project under the default C:\Users\<user>\source\repos path, I get the "MDK Deploy Script" option when right clicking on the project in the solution explorer.

However, when I use a different path, there is no deploy option. And if I manually add the "MDK Deploy All Scripts" command on my toolbar and run it, it shows a dialog saying 

    Deployment Cancelled

    There were no deployable scripts in this solution.

An example of the exact paths that worked and didn't work are:

(works)
C:\Users\jesusfreke\source\repos\SpaceEngineers\LargeMinerScript

(doesn't work)
C:\Users\jesusfreke\projects\SpaceEngineers\LargeMinerScript

**Comment by malware-dev** [comment]
I don't know. _All_ my projects are in non-standard locations.

**Comment by malware-dev** [comment]
If MDK doesn't recognize a project, it's because it's not finding key files under the projects' MDK folder.

**Comment by JesusFreke** [comment]
Yeah, I dunno. I followed *exactly* the same steps in both cases. The only thing different is that I chose a different path.

**Comment by JesusFreke** [comment]
Yeah, I dunno. I followed *exactly* the same steps in creating the two projects. The only difference is the location I chose to save them in. 

I just tried copying the working project to the same non-working path, and I still get the same issue when opening it from that path.

I also tried copying it to a different path (C:\Users\jesusfreke\SpaceEngineers\LargeMinerScript), and it *does* work. So there's something about my projects folder that it doesn't like.

**Comment by JesusFreke** [comment]
More troubleshooting:

I created a new projects2 folder, and copied the entire contents of projects to projects2. Opening the project from the new projects2 folder doesn't work.

I moved everything from projects2 back to projects, except the SpaceEngineers folder. Closed VS, reopened VS and the project, and it works.

I guess now I binary search *which* folder in my projects directory is responsible.
